swarm_copy/tools/bluenaas_memodel_getone.py

```diff
@@ -24,7 +24,7 @@ class MEModelGetOneMetadata(BaseMetadata):
 class InputMEModelGetOne(BaseModel):
     """Inputs for the BlueNaaS single-neuron simulation."""
 
-    model_id: str = Field(
```

Comment:
Same here, `simulation_id` is not a fitting name.

Reply:
I changed it to memodel id